src/lime/_internal/backend/native/NativeApplication.hx

```diff
@@ -746,7 +742,7 @@ class NativeApplication
 {
 	var AXIS_MOVE = 0;
 	var HAT_MOVE = 1;
-	var TRACKBALL_MOVE = 2;
+	// var TRACKBALL_MOVE = 2;
```

Comment:
Also this.

Reply:
Since these values are already pegged to constants, there is no reason to keep this here. I made a commit to remove it.

Reply:
I guess having a comment could explain why 2 is skipped, but there doesn't need to be. Skipping numbers is normal in enums, and usually unexplained.